VideoLocker/res/values/strings.xml

```diff
@@ -450,6 +448,8 @@
     <string name="title_lecture_complete">You\'ve reached the end of this video.</string>
     <!-- Alert dialog prompt at end of video. -->
     <string name="msg_lecture_complete">Would you like to watch the next video?</string>
+    <!-- Accessible content description of cancel button to cancel video download -->
+    <string name="cancel">Cancel</string>
```

Comment:
We already have a ``Cancel`` string - though maybe it needs to be renamed to be a little more generic.